disnake/enums.py

```diff
@@ -1605,156 +553,69 @@ def target_type(self) -> Optional[str]:
-    """The user is a Discord Certified Moderator."""
     http_interactions_bot = 524288
-    """The user is a bot that only uses HTTP interactions.
-
-    .. versionadded:: 2.3
-    """
     spammer = 1048576
-    """The user is marked as a spammer.
-
-    .. versionadded:: 2.3
-    """
 
 
 class ActivityType(int, Enum):
-    """Specifies the type of :class:`Activity`. This is used to check how to
-    interpret the activity itself.
-    """
-
     unknown = -1
-    """An unknown activity type. This should generally not happen."""
     playing = 0
-    """A "Playing" activity type."""
     streaming = 1
-    """A "Streaming" activity type."""
     listening = 2
-    """A "Listening" activity type."""
     watching = 3
-    """A "Watching" activity type."""
     custom = 4
-    """A custom activity type."""
     competing = 5
-    """A competing activity type.
-
-    .. versionadded:: 1.5
-    """
 
 
 class TeamMembershipState(int, Enum):
-    """Represents the membership state of a team member
-    retrieved through :func:`Client.application_info`.
-
-    .. versionadded:: 1.3
-    """
-
     invited = 1
-    """Represents an invited member."""
     accepted = 2
-    """Represents a member currently in the team."""
 
 
 class WebhookType(int, Enum):
-    """Represents the type of webhook that can be received.
-
-    .. versionadded:: 1.3
-    """
-
     incoming = 1
-    """Represents a webhook that can post messages to channels with a token."""
     channel_follower = 2
-    """Represents a webhook that is internally managed by Discord, used for following channels."""
     application = 3
-    """Represents a webhook that is used for interactions or applications.
-
-    .. versionadded:: 2.0
-    """
 
 
 class ExpireBehaviour(int, Enum):
-    """Represents the behaviour the :class:`Integration` should perform
-    when a user's subscription has finished.
-
-    There is an alias for this called ``ExpireBehavior``.
-
-    .. versionadded:: 1.4
-    """
-
     remove_role = 0
-    """This will remove the :attr:`StreamIntegration.role` from the user
-    when their subscription is finished.
-    """
     kick = 1
-    """This will kick the user when their subscription is finished."""
 
 
 ExpireBehavior = ExpireBehaviour
 
 
 class StickerType(int, Enum):
-    """Represents the type of sticker.
-
-    .. versionadded:: 2.0
-    """
-
     standard = 1
-    """Represents a standard sticker that all Nitro users can use."""
     guild = 2
-    """Represents a custom sticker created in a guild."""
 
 
 class StickerFormatType(int, Enum):
-    """Represents the type of sticker images.
-
-    .. versionadded:: 1.6
-    """
-
     png = 1
-    """Represents a sticker with a png image."""
     apng = 2
-    """Represents a sticker with an apng image."""
     lottie = 3
-    """Represents a sticker with a lottie image."""
 
     @property
     def file_extension(self) -> str:
-        """The file extension associated with this type of sticker."""
```

Comment:
this one too

Reply:
Resolved in 606392a3f3350c8f1e22fa43c4b268855841d5cf.